source/includes/fact-verifier-limitations.rst

```diff
@@ -0,0 +1,41 @@
+
+Compatibility
+~~~~~~~~~~~~~
+
+The embedded verifier is not available in mongosync 1.8 and
+earlier.
+
+For alternative verification methods, see :ref:`c2c-verification`.
+
+Limitations
+~~~~~~~~~~~
+
+The embedded verifier has the following limitations:
+
+- ``mongosync`` stores the verifier state in memory, which can
```

Comment:
We had to drop support for verification on migrations with sharded clusters. Can we add this as the first limitation?

Reply:
Fixed